handlers/easy_info/__init__.py

```diff
@@ -1,10 +1,40 @@
 from eventhandler import EventHandler
 
+ASSIGN_MSG = '@highfive: assign me'
+
+MSG = ('Hi! If you have any questions regarding this issue, feel free to make'
+       ' a comment here, or ping us at `#servo` channel in IRC '
+       '(`irc.mozilla.org`)\n\n'
+       'If you intend to work on this issue, then add `%s`'
+       ' to your comment, and I\'ll assign this to you. :smile:') % ASSIGN_MSG
+
+RESPONSE_FAIL = ('This looks like it has already been assigned to someone.'
+                 'I\'ll leave the decision to a core contributor.')
+
+RESPONSE_OK = ('Hey @%s! Thanks for showing interest on this issue.'
+               'This is now assigned to you!')
```

Comment:
`Hey @%s! Thanks for your interest in working on this issue. It's now assigned to you!`